velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -94,6 +94,7 @@ class ReaderBase {
       uint32_t maxSchemaElementIdx,
       uint32_t maxRepeat,
       uint32_t maxDefine,
+      uint32_t parentSchemaIdx,
```

Comment:
@Yuhta Thanks for catching this! I forgot to change the type before I submitted the PR. However, I just changed the initial value from -1 to 0 to make the code simpler (no need to check if it's >=0). This is ok because it's never required to check the parent of the root in getParquetColumnInfo(). The root is always named "hive_schema" with annotation of "UTF8" and cannot be any of the MAP_KEY_VALUE, MAP, LIST converted types.

```
  // Setting the parent schema index of the root("hive_schema") to be 0, which
  // is the root itself. This is ok because it's never required to check the
  // parent of the root in getParquetColumnInfo().
  schemaWithId_ = getParquetColumnInfo(
      maxSchemaElementIdx, maxRepeat, maxDefine, 0, schemaIdx, columnIdx);
```